tools/prometheus-builder/Dockerfile

```diff
@@ -4,6 +4,8 @@ RUN mkdir -p /go/src/github.com
 
 COPY ./build.sh /go/src/github.com/build.sh
 
-RUN chmod +x /go/src/github.com/build.sh
+COPY ./key.sh /go/src/github.com/key.sh
+
+RUN chmod +x /go/src/github.com/*
```

Comment:
We should just give executable permission in this directory. This could create security issues. Just give all the known script names.